### PR TITLE
Create newfile

### PR DESCRIPTION
**Pull request by pandeyshradha**

### CODE REVIEW

**Comment by Subhamdasskd** [comment]
Understood nothing
And 
Hola Amigos

**Review by Shubhamkumar2727 — COMMENTED** [review]
ok

File: newfile [hunk 1]
```diff
@@ -0,0 +1 @@
+hi
```
Comment (Subhamdasskd) [inline newfile:1]: CAn you plz explain this

Comment (Shubhamkumar2727) [reply newfile:1]: no

**Pull request merged**